lib/src/mt_ptp.c

```diff
@@ -435,8 +434,11 @@ static void ptp_delay_req_task(struct mt_ptp_impl* ptp) {
     rte_memcpy(ipv4_hdr, &ptp->dst_udp, sizeof(*ipv4_hdr));
     ipv4_hdr->udp.src_port = htons(MT_PTP_UDP_EVENT_PORT);
     ipv4_hdr->udp.dst_port = ipv4_hdr->udp.src_port;
+    ipv4_hdr->udp.dgram_cksum = 0;
     ipv4_hdr->ip.time_to_live = 255;
     ipv4_hdr->ip.packet_id = htons(ptp->t3_sequence_id);
+    ipv4_hdr->ip.next_proto_id = 17;
```

Comment:
Using 'IPPROTO_UDP' rather than '17' is preferred.

Reply:
> Using 'IPPROTO_UDP' rather than '17' is preferred.

DONE.